src/components/GenreSelect/GenreSelect.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+const defaultSelected = "All";
+
+class GenreSelect extends Component {
+  state = {
+    genres: [],
+    selected: "",
+    onSelect: (genre) => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.state.genres = Array.from(props.genres) || new Array() < "" > 0;
```

Comment:
Why do you write this? 
`Array.from(props.genres) || new Array() < "" > 0;`

Reply:
It didn't accept _this.state.genres_ as Array, so map(...) was imposible. The right part of the expression was supposed to be new Array()<"">(0) with zero capacity, but something went wrong 😂. 
Simplified with "[]" (acb780a178648469d974bc88446542caddf0576e)